content/en/blog/memory_management/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Memory management techniques"
+date: 2022-05-20T08:00:00Z
+draft: false
+contributors: [Cloud Hypervisor Team]
+---
+
+Depending on the expectations around the workload running in a Virtual Machine
+(VM), as well as the agreement between a customer and a Cloud Service Provider
+(CSP), multiple use cases related to memory management between host and guest
+arise and they can be addressed through different techniques.
+
+### Host managed
+
+This section is about how memory can be managed by the host, as it knows the VM
+is going to need more or less memory during its runtime.
+
+This is particularly well illustrated by the Cloud Native use case, involving
+containers to be created or destroyed, leading to the need to adjust the amount
+of memory available.
+
+We can also take the example of a user running a single VM with a certain amount
+of RAM until he needs more because of a different type of workload that requires
+a larger amount of memory.
+
+For both of these use cases, the goal is to save resources and therefore money.
+You don't want to run a very large VM just in case you might need more memory
+later because the larger the VM, the more expensive the cost. And that's why
+having access to such flexibility is very convenient for our users.
+
+#### Ballooning
+
+This technique was the former way of performing some sort of hot-plug/unplug
+without dealing with the complexity associated with adding or removing memory
+slots. The logic is reversed in a sense, as the VM is created with the maximum
+amount of memory that can be expected, and a balloon is inflated inside the
+guest, in order to reduce the available amount of memory.
+
+From a host perspective, if the system is running low on memory, inflating the
+balloon from one or several VMs can help the host getting some memory back to
+keep the system afloat.
+On the other hand if the host has enough spare memory to give some more to one
+of its VMs, it can deflate the balloon.
+
+I really want to emphasize on the fact that until recently (introduction of

```

Comment:
This is the only place you express a personal emphasis. Would be better to express this in a more abstract 3rd person e.g. "It is important to note that until recently..."